src/app/users/users-profile/users-profile-dialog.component.ts

```diff
@@ -10,6 +11,7 @@ import { MAT_DIALOG_DATA } from '@angular/material';
       <planet-users-profile [isDialog]="true" [userName]="name" [planetCode]="planetCode"></planet-users-profile>
     </mat-dialog-content>
     <mat-dialog-actions>
+      <button color="primary" *ngIf="!isHidden" routerLink="/manager/users" mat-raised-button mat-dialog-close i18n>Manager Settings</button>
```

Comment:
This link should go to the specific member profile, not the list of all members.